docs/follower_node.md

```diff
@@ -0,0 +1,56 @@
+# Follower Node
+
+When started with `"EnableFollowMode": true`, algod starts with a special
+property which allows it to be paused. This allows software to be written
+which runs synchronously with a node. If you need account balances at each
+round, or at a particular round, this is the only way to do it. On start
+the node will be paused, the REST API is used to resume and select a new
+round.
+
+It also allows fetching a `Ledger State Delta` object for recent rounds.
+This object is not designed for external users, but may be useful for
+advanced applications if you're willing to figure things out. It contains
+complete information for transitioning a database to the next round,
+including new account balances, changes to application and asset states,
+and new box information. Such information was previously unavailable to
+application developers.
+
+## Configuration
+
+Several options in the `config.json` file are needed:
+
+| property | description |
+| EnableFollowMode | When set to `true` the node starts as a network follower. | 
+| MaxAcctLookback | The number of additional `Ledger State Delta` objects available. |
+| CatchupParallelBlocks | This is useful for performance tuning. |
+
+## Usage
+
+New public endpoints are available to control the node:
+* `GET /v2/ledger/sync` - fetch the current minimum sync round.
+* `DELETE /v2/ledger/sync` - resume normal catchup by deleting the sync round.
+* `POST /v2/ledger/sync/{round}` - set the sync round.
+
+The `Ledger State Delta` object is not designed for external consumption,
```

Comment:
maybe we should mention conduit here.

Reply:
(it's not mentioned anywhere)

Reply:
or/also maybe in the intro, now that I think about it

Reply:
I didn't want to get into Conduit here, it's a client of this interface.

Reply:
well, the whole point of follower node is Conduit... you say "This object is not designed for external users, but may be useful for advanced applications if you're willing to figure things out." so you do talk about what it's _not_ designed for, why not talk about what it is designed for? As it's written now if someone reads this they're going to wonder what the heck it's for, since your statement is not inviting anyone to use it. I think you could add some brief mention and link to Conduit without getting into it.

Reply:
Hmm, I'm not sure. For some reason, it felt out of place to mention it. I added it to the top saying this mode was was created to be Conduit's data source.